turtlebot_navigation/CMakeLists.txt

```diff
@@ -21,6 +21,43 @@ include_directories(${catkin_INCLUDE_DIRS})
 add_executable(laser_footprint_filter src/laser_footprint_filter.cpp)
 target_link_libraries(laser_footprint_filter ${catkin_LIBRARIES})
 
+# Generate camera param files
+include($ENV{ROS_ROOT}/core/rosbuild/rosbuild.cmake)
```

Comment:
Yes, that is the only reason it is included.  Is there a better/modern way to achieve this?